npm_and_yarn/helpers/run.js

```diff
@@ -1,7 +1,5 @@
 #!/usr/bin/env node
 
-const functionMap = require("../lib");
```

Comment:
Keen to require the helper scripts explicitly here, something like:

```suggestion
const yarnHelpers = require(`./lib/yarn`);
const npmHelpers = require(`./lib/npm`);
const functionMap = { ...yarnHelpers, ...npmHelpers };
```